Add client details

hihuz opened the pull request:
This PR adds some components to show client details on click in the user dashboard.

After this, only will remain:
* client deletion confirmation through a dialog box (mutation also needs to be written)
* client update page needs to be stitched together reusing client creation components (should be pretty fast)

I wanted to add a transition when user opens the details but really didn't have the time, looks like this so far:
<img width="718" alt="Screenshot 2019-10-24 at 18 13 15" src="https://user-images.githubusercontent.com/3253522/67504908-8cd1f300-f68a-11e9-8ce8-b0fe5f5a6455.png">

and for small device:
<img width="381" alt="Screenshot 2019-10-25 at 09 48 36" src="https://user-images.githubusercontent.com/3253522/67553073-aa47a100-f70c-11e9-83fc-26223081bcd5.png">

hihuz committed:
Display client details on toggle

hihuz committed:
Add actions components to client details

hihuz committed:
Improve styles for small devices

johannespfeiffer reviewed (approved):
Great work!

1. Please add a bit of space here (see screenshot)
<img width="380" alt="Bildschirmfoto 2019-10-25 um 10 02 25" src="https://user-images.githubusercontent.com/6780316/67553960-b03e8180-f70e-11e9-990f-a8cd65adc369.png">

2. Do we really need to use a password field for the secret? I think in a later iteration we might provide a random string and make it readonly instead.

hihuz committed:
Improve spacing for client details

hihuz committed:
Use text field for secret

hihuz commented:
Thanks @johannespfeiffer !

1. good catch, I have added some padding:
<img width="709" alt="Screenshot 2019-10-25 at 10 24 04" src="https://user-images.githubusercontent.com/3253522/67555636-f517e780-f711-11e9-8a0c-966cd8ea3dbb.png">

2. Yeah I agree, I have changed it to plain text for now

hihuz commented:
Had to rebase to fix a conflict 🙂
